cmd/csi-sanity/Makefile

```diff
@@ -1,5 +1,5 @@
 APP_NAME := csi-sanity
-VER :=$(shell git describe)
+VER :=$(shell git describe --tags --dirty)
```

Comment:
Due to some reason, just `describe` was returning v0.2.0 tag for me. After setting `--tags`, it returned v0.3.0. And `--dirty` is nice to have :)
Hope it's fine to have these.